docs/source/features.rst

```diff
@@ -48,6 +48,32 @@ can be accessed under `/apis/swagger/schema/swagger-ui/#/apis/apis_api_history_e
 .. _django-simple-history: https://django-simple-history.readthedocs.io/en/latest/
 
 
+Management Commands
+^^^^^^^^^^^^^^^^^^^
+
+The APIS history plugin is based on the `django-simple-history` package. It comes with 
+management commands provided by `django-simple-history`. Some of the more useful commands are:
+
+.. code-block:: bash
+
+  python manage.py populate_history --auto
+
+This command will populate the history tables with the current state of the models. This is useful 
+if you have added the `VersionMixin` to an existing model.
+
+.. code-block:: bash
+
+  python manage.py clean_duplicate_history --auto
+
+This command will clean up duplicate history entries.
```

Comment:
Could there be a 1-sentence or half sentence explanation about duplicates added?

Why are there duplicates/how do they come about/why do they need cleaning? Or alternatively a link to where the django-simple-history docs talk about them.

Reply:
I dont think that there is an answer to this. Depending on your code you might call save on an object twice (eg via signals) and get duplicates. This management command just checks whether there are exactly the same objects in your history tables and removed one.